research/frontend/sign-up.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css">
+    <link rel="stylesheet" type="text/css" href="css/sign-up.css">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
+    <div class="container ">
+        <div class="card-panel z-depth-4 center">
+            <h4 class="header2">Register</h4>
+            <div class="row">
+                <form id="sign-up-form" class="col s12" method="post">
+                    <div class="row">
+                        <div class="input-field col s6">
+                            <input id="first_name" type="text">
```

Comment:
no closing tag? or <input xxxx /> will do

Reply:
do you mean the input tag?

Reply:
oh my bad. You only need closing tag for XHTML. In HTML, <input> tag has no end tag :)

Reply:
Oh i see! Thanks u sooooo much >_<